chain-impl-mockchain/src/certificate/vote_tally.rs

```diff
@@ -56,9 +69,27 @@ impl VoteTally {
         self.payload.payload_type()
     }
 
+    pub fn decryption_shares(&self) -> Option<&TallyDecryptShares> {
```

Comment:
Bikeshed: is it "decrypted shares", "decrypt-shares", or "decryption shares"? We should settle on the most meaningful naming variant and use it consistently for greppability.